cannot create 999-snapshot project

### Describe the bug

using:

```
 quarkus --version
Client Version 2.1.3.Final
```

running:
```
 quarkus create -P 999-SNAPSHOT ss2
Creating an app (default project type, see --help).
[ERROR] ❗  Unable to create project: Failed to resolve the default platform JSON descriptor
```

should I not get a 999-SNAPSHOT based project or is sometihng else needed ?

### Expected behavior

_No response_

### Actual behavior

_No response_

### How to Reproduce?

_No response_

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

_No response_

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

_No response_

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

_No response_

--- a/devtools/cli/src/main/java/io/quarkus/cli/common/TargetQuarkusVersionGroup.java
+++ b/devtools/cli/src/main/java/io/quarkus/cli/common/TargetQuarkusVersionGroup.java
@@ -8,6 +8,8 @@ import picocli.CommandLine;
 import picocli.CommandLine.Model.CommandSpec;
 
 public class TargetQuarkusVersionGroup {
+    final static String FULL_EXAMPLE = ToolsConstants.DEFAULT_PLATFORM_BOM_GROUP_ID + ":"
+            + ToolsConstants.DEFAULT_PLATFORM_BOM_ARTIFACT_ID + ":2.2.0.Final";
     StreamCoords streamCoords = null;
     String validStream = null;
 
@@ -18,7 +20,7 @@ public class TargetQuarkusVersionGroup {
     CommandSpec spec;
 
     @CommandLine.Option(paramLabel = "platformKey:streamId", names = { "-S",
-            "--stream" }, description = "A target stream, for example:%n  io.quarkus.platform:999-SNAPSHOT%n  io.quarkus.platform:2.0")
+            "--stream" }, description = "A target stream, for example:%n  io.quarkus.platform:2.0")
     void setStream(String stream) {
         stream = stream.trim();
         if (!stream.isEmpty()) {
@@ -34,7 +36,12 @@ public class TargetQuarkusVersionGroup {
     }
 
     @CommandLine.Option(paramLabel = "groupId:artifactId:version", names = { "-P",
-            "--platform-bom" }, description = "A specific Quarkus platform BOM, for example:%n  io.quarkus:quarkus-bom:2.0.0.Final")
+            "--platform-bom" }, description = "A specific Quarkus platform BOM, for example:%n"
+                    + "  " + FULL_EXAMPLE + "%n"
+                    + "  io.quarkus::999-SNAPSHOT"
+                    + "  2.2.0.Final%n"
+                    + "Default groupId: " + ToolsConstants.DEFAULT_PLATFORM_BOM_GROUP_ID + "%n"
+                    + "Default artifactId: " + ToolsConstants.DEFAULT_PLATFORM_BOM_ARTIFACT_ID + "%n")
     void setPlatformBom(String bom) {
         bom = bom.replaceFirst("^::", "").trim();
         if (!bom.isEmpty()) {

--- a/docs/src/main/asciidoc/cli-tooling.adoc
+++ b/docs/src/main/asciidoc/cli-tooling.adoc
@@ -173,12 +173,12 @@ Both `quarkus create` and `quarkus extension list` allow you to explicitly speci
 
 1. Specify a specific Platform Release BOM
 +
-A https://quarkus.io/guides/platform#quarkus-platform-bom[Quarkus Platform release BOM] is identified by `groupId:artifactId:version` (GAV) coordinates. When specifying a platform release BOM, you may use empty segments to fallback to default values (groupId: `io.quarkus`, artifactId: `quarkus-bom`, version: cli version). If you specify only one segment (no `:`), it is assumed to be a version.
+A https://quarkus.io/guides/platform#quarkus-platform-bom[Quarkus Platform release BOM] is identified by `groupId:artifactId:version` (GAV) coordinates. When specifying a platform release BOM, you may use empty segments to fallback to default values (shown with `quarkus create app --help`). If you specify only one segment (no `:`), it is assumed to be a version.
 +
 For example:
 +
-- Given the `2.0.0.Final` version of the CLI, specifying `-P :quarkus-bom:` is equivalent to `-P io.quarkus.platform:quarkus-bom:2.0.0.Final`.
-- Specifying `-P 999-SNAPSHOT` is equivalent to `-P io.quarkus:quarkus-bom:999-SNAPSHOT`
+- With the `2.0.0.Final` version of the CLI, specifying `-P :quarkus-bom:` is equivalent to `-P io.quarkus:quarkus-bom:2.0.0.Final`. Specifying `-P 999-SNAPSHOT` is equivalent to `-P io.quarkus:quarkus-bom:999-SNAPSHOT`.
+- With the `2.1.0.Final` version of the CLI, `io.quarkus.platform` is the default group id. Specifying `-P :quarkus-bom:` is equivalent to `-P io.quarkus.platform:quarkus-bom:2.1.0.Final`. Note that you need to specify the group id to work with a snapshot, e.g `-P io.quarkus::999-SNAPSHOT` is equivalent to `-P io.quarkus:quarkus-bom:999-SNAPSHOT`.
 +
 Note: default values are subject to change. Using the `--dry-run` option will show you the computed value.
 

